mirror-wordpress-plugins

```diff
@@ -124,6 +98,9 @@ class PluginUpdater
     extract_zip(@zip_file, Dir.pwd)
     puts "==> Committing and tagging the repo with full tag..."
     commit_and_tag
+    clean_local_files
+    verifier = PluginVerifier.new(@slug, @latest_wordpress_version, @full_path_to_clone)
```

Comment:
This isn't an issue introduced here, but I found the `latest_wordpress_version` variable name quite confusing, as for a while I thought it referred to the latest version of WordPress core, not the latest version of a particular plugin on wordpress.org